website/docs/BoilerplateProjects.md

```diff
@@ -253,14 +253,17 @@ Boilerplate project to run appium tests for native and mobile browser using moch
     - Contains sample test scenarios in mocha
     - Integrated with eslint and allure reporting
 
-### [17thSep/Mock-and-Spies](https://github.com/17thSep/MonitorNetworkLogs/)
+## [17thSep/WebdriverIO_Master](https://github.com/17thSep/WebdriverIO_Master)
 
-Boiler plate project for:
-
-- Capture Network Logs
-- Capture all GET/POST calls or a specific REST API
-- Assert Request parameters
-- Assert Response parameters
-- Store all the response in a separate file
+Boilerplate project to run **WebdriverIO v7** tests for the following:
 
-You can watch it created [on this video](https://www.youtube.com/watch?v=Be9IPyxHmLs).
+[WDIO 7 scripts with TypeScript in Cucumber Framework](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Cucumber)
+[WDIO 7 scripts with TypeScript in Mocha Framework](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Mocha)
+[Run WDIO 7 script in Docker](https://github.com/17thSep/WebdriverIO_Master/tree/master/TypeScript/Docker)
+[Network logs](https://github.com/17thSep/MonitorNetworkLogs/)
```

Comment:
This doesn't seem to be within the other repositories, maybe split it?

Reply:
I think it doesn't matter.

Reply:
fine

Reply:
I am not quite following Christian, what I was thinking is to have a single repository for all that I am planning to work on and create separate folders in it to segregate. That way I don't have to update multiple repositories separately making this page so long to scroll. 
Are you asking me to create different sections for each of the folders here?